plugins/generic/driver/locale/fr_CA/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/fr_CA/locale.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
All copyrights should be to Simon Fraser University, not Simon Fraser University Library; also update all copyright dates to end in 2018.